packages/design-system/src/styles/components/_Review.scss

```diff
@@ -0,0 +1,17 @@
+@import '../settings/index.scss';
+
+.ds-c-review {
+  border-bottom: 2px solid $border-color;
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+  padding-top: 16px;
+}
+
+.ds-c-review__heading {
+  display: inline-block;
+  font-weight: $font-bold;
+  font-size: $base-font-size;
+  margin-bottom: 0;
+  margin-top: 0;
```

Comment:
does this margin-top need to be here too? I dont see that originally in the utility classes

Reply:
oh i see is this from the `ds-text`?

Reply:
It's from `ds-u-margin-bottom--0`